recipes/mpir/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set version = "2.7.2" %}
+
+package:
+  name: mpir
+  version: {{ version }}
+
+source:
+  fn: mpir-{{ version }}.tar.bz2
+  url: http://mpir.org/mpir-{{ version }}.tar.bz2
```

Comment:
Need some kind of checksum here. Either md5 or some sha*. Possibly some other checksum could be used, as well.